src/schema_compile.c

```diff
@@ -1255,22 +1253,27 @@ lys_compile_unres_depset(struct ly_ctx *ctx, struct lys_glob_unres *unres)
     struct lysc_node *node;
     struct lysc_type *typeiter;
     struct lysc_type_leafref *lref;
-    struct lysc_ctx cctx = {0};
+    struct lysc_ctx cctx;
     struct lys_depset_unres *ds_unres;
+    struct ly_path *path;
     LY_ARRAY_COUNT_TYPE v;
-    uint32_t i;
+    uint32_t i, processed_leafrefs = 0;
 
     ds_unres = &unres->ds_unres;
 
     /* fake compile context */
+again:
+    cctx = (struct lysc_ctx) {
```

Comment:
Please, no constructs like this, use `memset` instead.

Reply:
ok